temp.py

```diff
@@ -0,0 +1,2 @@
+import pdb
```

Comment:
**CODE_QUALITY**: Avoid leaving debugging statements like `import pdb` in the production code. Remove this line before merging.
```python
# import pdb
```